scripts/reset.js

```diff
@@ -11,8 +11,8 @@ const cleaningProcess = spawn('git', [
   'clean',
   '-xdf',
   '-n',
-  '--exclude=".vscode"',
-  '--exclude=".idea"',
+  '--exclude="/.vscode"',
```

Comment:
@okonet Any idea why this is necessary? This seems to be more restrictive than what was there before

Reply:
Without that it would try to match `.vscode` recursively which would increase the amount of directories it would output dramatically because `.vscode` is often present in `node_modules`. With the `/` it would only match the root directories.

Reply:
I don't even think `/.vscode` is needed here since it's under VCS but I left it in case it contains some user-generated settings.

Reply:
Fair enough, but I was more interested in `'--exclude=".idea"',` which already existed in the code and was replaced by `--exclude="/.idea"` which seems more restrictive

Reply:
That's my question. Any idea why `".idea"` doesn't work but `"/.idea"` does?

Reply:
Not really. I would assume it could treat the . as part of regex but I didn't want to spend too much time investigating.